Documentation/Versioning.md

```diff
@@ -0,0 +1,62 @@
+# Azure SDK .NET - Versioning
+
+This document covers the basic versioning strategy and which properties to use. It is based on the rules outlined in our [Azure SDK Releases doc](https://github.com/Azure/azure-sdk/blob/master/docs/engineering-system/releases.md#net) and it closely matches the [.NET versioning rules](https://github.com/dotnet/arcade/blob/master/Documentation/CorePackages/Versioning.md) but simplified to only include the parts necessary for our libraries.
+
+## Package Versioning
+
+Package version will look like:
+```
+MAJOR.MINOR.PATCH-PRERELEASE
+```
+
+In our project file you we will use the `VersionPrefix` property to define the `MAJOR.MINOR.PATCH`
+
+```
+<VersionPrefix>1.0.0</VersionPrefix>
+```
+
+`PRERELEASE` will be controlled by:
+
+BuildNumber: Should be in the format `yyyyMMdd.<build revision>` and defaults to today's date `yyyyMMdd.1` but can be set by passing in and msbuild property for `OfficialBuildId` with a matching format.
```

Comment:
Should the default consider a time component as well, just in case multiple releases are performed in a day?

Reply:
We don't consider the time component but that is what the `<build revision>` component is for as it will have enable us to release as many times as we need to in a given day.

Reply:
Gotcha.... I missed the build identifier component and thought the `1` was static.   By bad.